frontend/packages/shared/src/api-paths/index.js

```diff
@@ -36,11 +36,11 @@ export const gitCommitPath = (org, app, commitId) => `/repos/${org}/${app}/commi
 export const repositoryGitPath = (org, app) => `/repos/${org}/${app}.git`;
 export const repositoryPath = (org, app) => `/repos/${org}/${app}`;
 export const repositoryOwnerPath = (org) => `/repos/${org}`;
-export const repositoryBasePath = () => '/repos';
-export const userLogoutPath = () => '/repos/user/logout';
+export const repositoryBasePath = () => "/repos";
```

Comment:
Hvorfor har noen fått doble fnutter, mens andre ikke?

Reply:
Ser ut som det er konsekvent men de som ikke er doble fnutter er "back-ticks" for å kunne skrive variabler i string'en. :) Jeg trodde for øvrig at vi brukte enkel fnutter istedenfor doble i prosjektet? :D

Reply:
Godt spotta... 

Trur prettier-config ikke funker på den måten jeg trodde. Altså at den resetter hele greia. Skal gjøre ett par tester.